### ISSUE TITLE
Tag is always vnull

### ISSUE THREAD

**Issue by AlexanderNZ** [mathieudutour/github-tag-action#21]
Hi there,

I've got this action in use in the following way, I'm trying to create a tag based on commit messages like this one:

`fix(prod-releases): uses the output of the tagging process as the input of the release phase`

And here's my action in use:

```
- name: Bump version and push tag
  id: create_tag
  uses: mathieudutour/github-tag-action@v4.5
  with:
    github_token: ${{ secrets.GITHUB_TOKEN }}
    release_branches: dev
```

And here's the output:

```
Run mathieudutour/github-tag-action@v4.5
/usr/bin/git fetch --tags
/usr/bin/git tag
refs/heads/dev
refs/pull/44/merge
v1.0.0
v1.1.0
v1.2.0
vnull
/usr/bin/git rev-list --tags --topo-order --max-count=1
673a23637907bac23d3998cbdcb5b98a8751d106
/usr/bin/git describe --tags 673a23637907bac23d3998cbdcb5b98a8751d106
vnull
/usr/bin/git log vnull..HEAD --pretty=format:'%s%n%b|commit-hash:%h==============================================' --abbrev-commit
'fix(prod-releases): uses the output of the tagging process as the input of the release phase
|commit-hash:933de6a=============================================='::debug::Setting previous_tag to: vnull
Analyzing commit: fix(prod-releases): uses the output of the tagging process as the input of the release phase
The release type for the commit is patch
Analysis of 1 commits complete: patch release
/usr/bin/git tag -l vnull
vnull
```

Not sure what I should do about this. Any thoughts?

**Comment by mm4492** [comment]
I have the same issue.

**Comment by boxcee** [comment]
Had some similar problems. Trying to fix some here: https://github.com/mathieudutour/github-tag-action/pull/36

**Comment by boxcee** [comment]
I think it should be solved with #37. Tags are filtered. Only valid tags will be used as 'starting point' to increment version.

**Comment by mathieudutour** [comment]
going to close this then!
